### ISSUE TITLE
ETA for v2.6?

### ISSUE THREAD

**Issue by viking2** [bkury/skin.night#28]
Hi Bkury,
I had hoped v2.6 would be out by now, but I guess it has been delayed? Any new ETA?
TIA!

p.s. Your Night Skin makes Kodi a much better product. I have eagerly awaited v 2.6!

**Comment by bkury** [comment]
Thanks!

The plan was to release v2.6 before finishing my KODI addon. Unfortunately, I realized too late that this would result in much more work afterwards, so I decided to finish my addon first.

The good news is that I'm almost done and that this addon will open up much more possibilities and features.
(Multiple ratings, Additional TV/Movie Information, Cast/Crew Images, TV Themes, News and much more ...)

ETA: ~4 weeks
(Please note that this could change again if something goes wrong)

Latest Screenshot / WIP:
![screenshot](https://user-images.githubusercontent.com/1777146/137754973-033d2c6c-f481-43d8-b657-7990abb62af3.jpg)

**Comment by viking2** [comment]
Sounds great!
Here is are some of the important things that I am looking forward to:
1. More obvious highlight of playback items (see https://github.com/bkury/skin.night/issues/24)
2. See if video has subtitles available (see https://github.com/bkury/skin.night/issues/25)
3. Increased font size (see https://github.com/bkury/skin.night/issues/22)
4. Mark as watched/unwatched (see https://github.com/bkury/skin.night/issues/21)
5. Better Subtitle Search Results (see https://github.com/bkury/skin.night/issues/29)

EDIT: Added number 5 above

**Comment by viking2** [comment]
Hi Bkury,
Are you still working on v.2.6?
TIA!

**Comment by bkury** [comment]
`Are you still working on v.2.6?`


Sure! Unfortunately, I really underestimated the time it takes me to rewrite some of the older skin code which is necessary for going forward. Some other personal work stuff didn't help either ...

Still trying to get the update out as soon as possible!

**Comment by viking2** [comment]
Understood!
Biting my nails, almost nothing left :wink:

**Comment by viking2** [comment]
Hi Bkury,
Are you still working on v.2.6..?
TIA!

**Comment by bkury** [comment]
Yes, I'll post a general update / screenshots next week (kodi forum). 

Also, a test/pre-release will be available by the end of this month.

**Comment by viking2** [comment]
Great! Can't wait :thumbsup: :smiley: !

**Comment by viking2** [comment]
> Yes, I'll post a general update / screenshots next week (kodi forum).
> 
> Also, a test/pre-release will be available by the end of this month.

End of _**this**_ month ... 😉?

**Comment by bkury** [comment]
Sorry, for the short delay but this project got even more time-consuming than I ever expected.

But I can assure you, it's not that far off anymore.

**Comment by viking2** [comment]
Hi, 4 months later. Any update.....?

**Comment by bkury** [comment]
Hi! I have some spare time in december, will post an update before christmas.

**Comment by viking2** [comment]
Great, it will be a nice Christmas present for us  :-)!

**Comment by viking2** [comment]
Is it Christmas yet  😉 ?
p.s. Happy New Year!

**Comment by viking2** [comment]
Hi bkury,
Did you give up on v2.6?

**Comment by viking2** [comment]
Hi again bkury,

I haven't heard back from you for almost 5 months. I guess that you never had time to finish v2.6.

I wonder if you are aware of an alternative to your Night skin? I haven't found anything that comes even close.
TIA!

**Comment by bkury** [comment]
Hi!

Sorry, I didn't get a notification for your comment in Feb but I have posted an update on KODIs forum a while ago.

I haven't given up on v2.6, a ton of work is already done and everything is slowly coming together. 
I can't give an exact ETA but I'm still aiming for end of May.

**Comment by viking2** [comment]
Sounds great!
I recently tried to check the KODI forum, but it has been down since it was breached  ;-(.

BTW, I hope that NIGHT v2.6 will be compatible with KODI v19 & v20?
I am still on KODI v18.9 as I didn't want to upgrade until the new Night skin was out....

**Comment by bkury** [comment]
> BTW, I hope that NIGHT v2.6 will be compatible with KODI v19 & v20?

Yes, it will support KODI v18 - v21 and my own KODI fork "OpenHT v20" with some additional features.

**Comment by viking2** [comment]
> Hi!
> 
> Sorry, I didn't get a notification for your comment in Feb but I have posted an update on KODIs forum a while ago.
> 
> I haven't given up on v2.6, a ton of work is already done and everything is slowly coming together. I can't give an exact ETA but I'm still aiming for end of May.

Any new ETA..?

**Comment by viking2** [comment]
Hi bkury,
Are you still working on this or was it too much work to complete?
Thanks!
